### ISSUE TITLE
SavedModel: enable dropout & disable batch normalization

### ISSUE THREAD

**Issue by zndr27** [tensorflow/tensorflow#60640]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Support

### Have you reproduced the bug with TF nightly?

Yes

### Source

binary

### Tensorflow Version

2.14.0-dev

### Custom Code

Yes

### OS Platform and Distribution

Linux Ubuntu 20.04

### Python version

3.11

### Current Behaviour?

I am currently training a ResNet model with both batch normalization and dropout layers. My goal is to use monte carlo dropout for uncertainty estimation at evaluation time (i.e. with training=False in my model call).

I'm currently working with tf.functions and the SavedModel format (not eager execution). Eager execution is to slow for my application, and therefore is not an option for me. 

Thus, when I set training=False for my model calls, it disables both the batch normalization and dropout layers. However, when I set training=False I want to keep dropout enabled but disable batch normalization (for the purpose of uncertainty estimation).

How can I achieve this with tf.function and the SavedModel format?

I am using tf-nightly.

### Standalone code to reproduce the issue

```shell
n/a
```

### Relevant log output

_No response_</details>

**Comment by zndr27** [comment]
Bump

**Comment by SuryanarayanaY** [comment]
Hi @zndr27 ,

I need to check which Resnet model you have used. Did you used model from `tf.keras.applications.resnet` ?

I am attaching the source [code](https://github.com/keras-team/keras/blob/v2.12.0/keras/applications/resnet.py#L536-L568) of Resnet from the API `tf.keras.applications.resnet` . Please have a look and confirm whether it helps.

I quickly gone through the source code of `Resnet` and as per my understanding `batch normalization` implemented for all the cases. I think you may customize the code locally for your requirement

Thanks!
